contracts/arbitrators/KlerosLiquidProxy.sol

```diff
@@ -92,6 +92,11 @@ contract KlerosLiquidProxy is Multicall, IKlerosLiquidProxy {
             claimsManager.isManagerOrMediator(msg.sender),
             "Sender cannot mediate"
         );
+        (, address arbitrated, , , , , , bool ruled) = IKlerosLiquid(
+            address(klerosArbitrator)
+        ).disputes(disputeId);
+        require(arbitrated == address(this), "Invalid dispute ID");
+        require(!ruled, "Dispute already ruled");
```

Comment:
Could it be that it was never ruled but the voting period has ended? If so then we might want to retrieve [period](https://github.com/kleros/kleros/blob/master/contracts/kleros/KlerosLiquid.sol#L83) from the dispute and validated against `block.timestamp` 🤔

Reply:
Could be, I'll look into it

Reply:
Fixed in https://github.com/api3dao/claims-manager/pull/54/commits/de3065efcf493c25b4d0fd69d8e445b30ae5d9f5
The dispute goes into the evidence period after it is first created and after each appeal, it looks like it is intended for evidence to be submitted then